requirements/django22.txt

```diff
@@ -1,4 +1,3 @@
 Django>=2.2,<2.3
-django-filter>=2
 django-crispy-forms>=1.7,<1.8
```

Comment:
For a second I though that crispy-forms* missing from the 3.2 reqs was a problem, but then I remembered that we are not testing on 3.2 at this time, and the crispy-forms* lines are moved to the base requirements when we do.